tests/unit/test_path_info.py

```diff
@@ -85,3 +85,21 @@ def test_https_url_info_str():
 def test_path_info_as_posix(mocker, path, as_posix, osname):
     mocker.patch("os.name", osname)
     assert PathInfo(path).as_posix() == as_posix
+
+
+@pytest.mark.parametrize("cls", [HTTPURLInfo])
```

Comment:
No need for this, we could just use `HTTPURLInfo` directly in the test.

Reply:
okey, I just followed the pattern from previous tests.